src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,44 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        Report report = readReportDataFromFile(fromFileName);
+        writeReportToFile(toFileName, report.createReport().toString());
+    }
+
+    private void writeReportToFile(String fileName, String report) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(fileName))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fileName, e);
+        }
+    }
 
+    private Report readReportDataFromFile(String fileName) {
```

Comment:
you shouldn't read from file and get created report here, this method should only read from file